### PR TITLE
Lstm

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

**Comment by tomkoren21** [comment]
@doronHarnoy @tip3x  PTAL. specifically look at the dynamic reshape and slice dynamic shapes

File: onnx2kerastl/reshape_layers.py [hunk 1]
```diff
@@ -202,20 +202,31 @@ def target_layer(x):
...
+                input_0_shape = input_0.shape
+                first_mismatch = np.argmin(np.array(input_0_shape[:len(input_1)]) == input_1)
+                if (input_1 == None).any() and (np.array(input_0_shape) == None).any() and len(input_1) < len(input_0_shape)\
+                        and input_1[first_mismatch] == -1: #reshape end
+                    end_match_arr = np.array(input_0_shape[-len(input_1):]) == input_1
+                    end_idx_match = np.argmax((np.array(input_0_shape[-len(input_1):]) == input_1))
+                    end_idx_match = end_idx_match + len(input_0_shape) - len(input_1) if end_idx_match > first_mismatch \
+                                                     and end_match_arr[end_idx_match] else len(input_0_shape) + 1
+                    tf_shape = tf.shape(input_0)
+                    layers[node_name] = tf.reshape(input_0, [*tf_shape[:first_mismatch], -1, *tf_shape[end_idx_match:]])
```
Comment (tip3x) [inline onnx2kerastl/reshape_layers.py:214]: replace with `tf.reshape(input_0, input1)`

Comment (tomkoren21) [reply onnx2kerastl/reshape_layers.py:214]: Discussed this together (won't work in this way since input1 should be a kerastensor).

**Review by tip3x — APPROVED** [review]
LGTM

**Comment by tomkoren21** [comment]
Moving this to draft untill we complete the require testing and finalize the required LSTM work

**Pull request closed without merging**